routes/api.php

```diff
@@ -233,6 +234,9 @@
         });
 
         Route::get('user-lms-settings', 'UserLmsSettingsController@index');
+        // default Sso Integration Setting
+        Route::apiResource('default-sso-settings', 'DefaultSsoIntegrationSettingsController');
+        Route::get('organizations/searchOrganizationByName', 'OrganizationController@searchOrganizationByName')->name('searchOrganizationByName');
```

Comment:
Why are you using camel case for URLs?
Use PayPal API standards.
it should be ; organizations/search or something similar.